discretisedfield/region.py

```diff
@@ -746,7 +736,59 @@ def __truediv__(self, other):
         .. seealso:: :py:func:`~discretisedfield.Region.__mul__`
 
         """
-        return self * other ** (-1)
+        self._check_operand(factor, True, "operator /")
+        return self._apply_operator(factor, np.divide)
+
+    def scale(self, factor):
+        """Scale the region."""
+        self._check_operand(factor, True, "method scale")
+        return self._apply_operator(factor, np.multiply)
+
+    def __add__(self, vector):
+        """Translate the region."""
+        self._check_operand(vector, False, "operator +")
+        return self._apply_operator(vector, np.add)
+
+    def __radd__(self, vector):
+        return self + vector
+
+    def __sub__(self, vector):
+        """Translate the region."""
+        self._check_operand(vector, False, "operator -")
+        return self._apply_operator(vector, np.subtract)
+
+    def __rsub__(self, vector):
+        return self - vector
```

Comment:
Is this correct? Should it be something like ` -1 * self + vector`?

Reply:
You are right, this is not correct.

Maybe we should not implement `__radd__` and `__rsub__` (and maybe `__rmul__`). Thoughts @samjrholt ?